jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/DeploymentScanner.java

```diff
@@ -1484,7 +1480,7 @@ private static boolean isDeployableDirectory(Path p)
                 return false;
             if (Files.isDirectory(p))
             {
-                return !FileID.isExtension(p, "d"); // ignore nominated dirs
+                return !FileID.isExtension(p.toUri(), "d"); // ignore nominated dirs
```

Comment:
Why this double check?

I think the check should be done only in the filter, so here we are sure these directories never show up.

Reply:
Resolved as well.